README.md

```diff
@@ -18,7 +18,65 @@ with Bazel.
 
 ## Setup
 
-To be documented.
+### Using npm and Node.js rules
+
+We recommend you use npm to install the PostCSS build rules if you use the
+[Node.js build rules][rules_nodejs], and **strongly recommend** using npm
+if you also depend or plan to depend on PostCSS in your local `package.json`.
+(For example, if PostCSS plugin(s) will form part of your repository's build
+pipeline.)
+
+Installing the PostCSS build rules via npm allows you to rely on its native
+package management functionality. If you depend on PostCSS in your local
+`package.json`, using npm also helps avoid version skew between PostCSS in
+your repository and these build rules.
+
+[rules_nodejs]: https://bazelbuild.github.io/rules_nodejs/
+
+Add the `@bazel/postcss` npm package to your `devDependencies` in
+`package.json`.
+
+Your `WORKSPACE` should declare a `yarn_install` or `npm_install` rule named
+`npm`. It should then install the rules found in the npm packages using the
+`install_bazel_dependencies' function.
+See https://github.com/bazelbuild/rules_nodejs/#quickstart
+
+This causes the `@bazel/postcss` package to be installed as a Bazel workspace
+named `build_bazel_rules_postcss`.
+
+Now add this to your `WORKSPACE` to install the PostCSS dependencies:
+
+```python
+# Fetch transitive Bazel dependencies of PostCSS. This is an optional step
+# because you can always fetch the required transitive dependencies yourself.
+load("@build_bazel_rules_postcss//:package.bzl", "rules_postcss_dependencies")
```

Comment:
same comment as below, can you drop this requirement? it causes eager install for builds that don't want to do css processing

Reply:
This refers to https://github.com/bazelbuild/rules_postcss/pull/42#discussion_r441634677, right? If so let's resolve this issue in #43.

Reply:
On closer look this can just be removed now